bin/main.py

```diff
@@ -8,29 +8,26 @@
 import logging
 import os
 import pandas
-import sys
-
-import spacy
 import textract
 
 import lib
 import field_extraction
 
-# Set encoding to utf8, to better deal w/ utf8 encoded resumes
-reload(sys)
-sys.setdefaultencoding('utf8')
-
+import en_core_web_sm
```

Comment:
This import isn't in the `environment.yaml` file. Could we update that file to include it?

`import en_core_web_md`

And change `environment.yaml` to read: 
```
name: resume
channels:
- !!python/unicode
  'defaults'
dependencies:
- !!python/unicode
  'openssl=1.0.2l=0'
- !!python/unicode
  'pip=8.1.2=py27_0'
- !!python/unicode
  'python=2.7.12=1'
- !!python/unicode
  'readline=6.2=2'
- !!python/unicode
  'setuptools=26.1.1=py27_0'
- !!python/unicode
  'sqlite=3.13.0=0'
- !!python/unicode
  'tk=8.5.18=0'
- !!python/unicode
  'wheel=0.29.0=py27_0'
- !!python/unicode
  'zlib=1.2.8=3'
- pip:
  - argcomplete==1.8.2
  - beautifulsoup4==4.5.3
  - boto==2.48.0
  - bz2file==0.98
  - certifi==2017.7.27.1
  - chardet==3.0.4
  - cymem==1.31.2
  - cytoolz==0.8.2
  - dill==0.2.7.1
  - docx2txt==0.6
  - ebooklib==0.15
  - en-core-web-md==1.2.0
  - ftfy==4.4.3
  - gensim==3.0.1
  - html5lib==0.999999999
  - idna==2.6
  - lxml==4.1.0
  - murmurhash==0.26.4
  - numpy==1.13.3
  - olefile==0.44
  - pandas==0.20.3
  - pathlib==1.0.1
  - pillow==4.3.0
  - plac==0.9.6
  - pocketsphinx==0.1.3
  - preshed==1.0.0
  - python-dateutil==2.6.1
  - python-pptx==0.6.5
  - pytz==2017.2
  - pyyaml==3.12
  - regex==2017.9.23
  - requests==2.18.4
  - scipy==0.19.1
  - six==1.10.0
  - smart-open==1.5.3
  - spacy==1.9.0
  - speechrecognition==3.6.3
  - termcolor==1.1.0
  - textract==1.6.1
  - thinc==6.5.2
  - toolz==0.8.2
  - tqdm==4.19.4
  - ujson==1.35
  - urllib3==1.22
  - wcwidth==0.1.7
  - webencodings==0.5.1
  - wrapt==1.10.11
  - xlrd==1.0.0
  - xlsxwriter==1.0.2
```

Reply:
I will revert this since the en_core_web_sm is not available in the conda or Pypi repository and can only be dowloaded via spacy (with the command: 'python -m spacy download en')